test/library/standard/FileSystem/chmod/chmod_noprivilege.chpl

```diff
@@ -0,0 +1,9 @@
+use FileSystem;
+// Test when the user tries to modify a file or a directory without privilege
+// Should throw PermissionError assuming user cannot modify root folder
+config const permissions = 0o777;
+config const original = 0o755;
```

Comment:
Now that we know the .preexec doesn't actually impact the .execopts, let's instead use [getMode](https://chapel-lang.org/docs/latest/modules/standard/FileSystem.html#FileSystem.getMode) to determine the value of `original`